app/payment-request-mapping/generate-invoice-number.js

```diff
@@ -0,0 +1,13 @@
+const generateInvoiceNumber = (paymentRequest) => {
+  const invoiceNumber = paymentRequest.invoiceNumber
+    ? paymentRequest.invoiceNumber.slice(-8)
+    : paymentRequest.agreementNumber.slice(-8)
+
+  const paymentRequestNumber = paymentRequest.paymentRequestNumber.toString().length === 1
+    ? `V0${paymentRequest.paymentRequestNumber}`
+    : `V${paymentRequest.paymentRequestNumber}`
```

Comment:
You could simplify this with `padStart()`.

eg

const paymentRequestNumber = `V${paymentRequest.paymentRequestNumber.toString().padStart(2,'0')}`